podcasts/FileManager+Helper.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+extension FileManager {
+
+    func fileSize(of url: URL) -> Int64? {
```

Comment:
You can use `url.resourceValues(forKeys: [.fileSizeKey]).fileSize` and avoid the `FileManager` here. I _think_ this is preferable since the API has slightly better type safety and the [FileManager docs](https://developer.apple.com/documentation/foundation/filemanager) say to avoid the string APIs:

> When specifying the location of files, you can use either NSURL or NSString objects. The use of the [NSURL](https://developer.apple.com/documentation/foundation/nsurl) class is generally preferred for specifying file-system items because URLs can convert path information to a more efficient representation internally.

Reply:
Will update the code to use that method instead.